meetings/conferences/serializers.py

```diff
@@ -1,22 +1,8 @@
-from rest_framework import serializers as ser
-from django_countries.fields import CountryField
+from rest_framework_json_api import serializers
 
 from conferences.models import Conference
 
-class ConferenceSerializer(ser.ModelSerializer):
-    country = CountryField() #get country_dict working later
-    start = ser.DateTimeField(source='event_start')
-    end = ser.DateTimeField(source='event_end')
-    submissionstart = ser.DateTimeField(source='submission_start')
-    submissionend = ser.DateTimeField(source='submission_end')
-    site = ser.URLField(required=False, allow_blank=True)
-    #url for now
-    logo = ser.URLField(required=False, allow_blank=True)
 
-    # Later on add tags and sponsors back
+class ConferenceSerializer(serializers.ModelSerializer):
     class Meta:
         model = Conference
-
-        fields = ('created', 'modified', 'id', 'title', 'logo', 'site', 'city',
```

Comment:
By default, I believe 'id' is special in drf and won't be auto included like we need to be for user to input conference slugs. http://stackoverflow.com/questions/15014495/id-field-in-django-rest-framework-serializer